ddpui/api/airbyte_api.py

```
@@ -1,7 +1,8 @@
 """Dalgo API for Airbyte"""
 
 import os
-from typing import List
+from typing import List, Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)